spec/Powertrain/TractionBattery/BatteryConditioning.vspec

```diff
@@ -0,0 +1,112 @@
+
+# New proposed signals for battery conditioning below
+#
+# Existing signals that may be of interest for battery conditioning:
+# Vehicle.Exterior.AirTemperature
+# Vehicle.LowVoltageSystemState
+# Vehicle.Powertrain.TractionBattery.Temperature.Average
+# Vehicle.Powertrain.TractionBattery.CurrentCurrent
+# Vehicle.Powertrain.TractionBattery.Charging.ChargeLimit
+# Vehicle.Powertrain.TractionBattery.Charging.MaximumChargingCurrent.*
+# Vehicle.Powertrain.TractionBattery.Charging.EvseId (to be able to know charger capabilities)
+
+############## END OF PART NOT TO BE MERGED ########################
+
+BatteryConditioning:
+  type: branch
+  description: Properties related to preparing the vehicle battery for charging or driving.
+
+BatteryConditioning.IsActive:
+  datatype: boolean
+  type: sensor
+  description: Indicates if battery conditioning is active (i.e. actively monitors battery temperature).
+               True = Active. False = Inactive.
+  comment: This signal is typically true when mode is not INACTIVE and time is within defined
+           start/end times.
+
+BatteryConditioning.IsOngoing:
+  datatype: boolean
+  type: sensor
+  description: Indicating if battery conditioning is currently ongoing.
+               Battery conditioning is considered ongoing when the battery conditioning system
+               is actively heating or cooling the battery, or requesting heating or cooling.
+  comment: When battery conditioning is active, but temperature is already within acceptable range so that
+           no cooling or heating is needed then IsOngoing shall report False.
+
+# Mode names aligned to indicate "target status" rather than "current status"
+# E.g. when parked you can theoretically prepare for both charging and driving
+
+# Also using an enum/allowed and not two boolean (IsFastChargingRequested,IsDrivingRequested) as
+# both modes cannot be active at the same time
+# An alternative approach could be to have a boolean "IsBatteryConditionRequested"
+# and then a separate enum (or boolean) indicate what type of charging preparation is requested
+
+BatteryConditioning.RequestedMode:
+  datatype: string
+  type: actuator
+  allowed: ['INACTIVE', 'FAST_CHARGING_PREPARATION', 'DRIVING_PREPARATION']
+  description: Defines requested mode for battery conditioning.
+               INACTIVE - Battery conditioning inactive.
+               FAST_CHARGING_PREPARATION - Battery conditioning for fast charging.
+               DRIVING_PREPARATION - Battery conditioning for driving.
+  comment: The Mode and End Time can be used to calculate target temperature and Start Time
+
+
+# Proposed to use unix time rather than time remaining (seconds) in VSS.
+# That does not prevent that time remaining is used internally,
+
+BatteryConditioning.StartTime:
+  type: actuator
+  datatype: uint64
+  description: Start time for battery conditioning, if Mode is not INACTIVE.
+  unit: unix-time
```

Comment:
Pierre Pierre brought up the topic of time representation. 

We now have two time units; `unix-time` (to be used with uint32/uint64/int64) and `iso8601` (to be used with string). Historically we have favored `iso8601` over unix time, but we do not have a deliberate decision. Technically we could use iso8601 here as well, so what do we prefer?

Reply:
I think we had several discussion wrt ti time https://github.com/COVESA/vehicle_signal_specification/issues/667 , https://github.com/COVESA/vehicle_signal_specification/issues/525  and favoring iso is a more "recent" historical decisions.

I think personally some of you guys convinced me to prefer iso instead of unix timestamps in the past :D In any case it is what we are recommending in the documentation: https://covesa.github.io/vehicle_signal_specification/rule_set/data_entry/data_types/#timestamps  and I would agree in terms of VSS data model (an implementation can still use faster/more primitive timestamps for internal representation)